### ISSUE TITLE
Syntax Coloring Stops working if 2d array is hit that is split across lines.

### ISSUE THREAD

**Issue by tmaslach** [jeff-hykin/cpp-textmate-grammar#510]
Checklist

- [X] This problem exists even with the setting `"C_Cpp.enhancedColorization": "Disabled"`
- [ ] This bug exists for C 
- [x] This bug exists for C++
- [ ] This bug exists for Objective-C
- [ ] This bug exists for Objective-C++

It really does look like it is only broken for C++!

## The code with a problem is:
```
// comment1 - nice and green
int ItIsAReallyLongVariableNameWith[A_REALLY_LONG_CONSTANT_NAME_HERE]
                                   [AND_A_REALLY_LONG_CONSTANT_NAME_HERE];

// comment2 - it's unfortunately white and everything below as well
typedef const int OtherStuff;

class Test
{
};
```

## It looks like:
The default Vscode theme is being used.
![image](https://user-images.githubusercontent.com/7267036/96306556-de773a00-0fcd-11eb-80ed-a719532de832.png)

## It should look like:
Moving the 2d array to be on one line completely fixes the problem.  Unfortunately, I need it split on 2 due to formatting requirements.
![image](https://user-images.githubusercontent.com/7267036/96306598-f9e24500-0fcd-11eb-8bae-69bc17b89bba.png)

**Comment by jeff-hykin** [comment]
Are you using the latest version of the extension? It works for me 

![Screen Shot 2020-10-18 at 9 45 22 PM](https://user-images.githubusercontent.com/17692058/96395850-52127600-118b-11eb-80d0-7f1893a64506.png)

**Comment by tmaslach** [comment]
Hi Jeff - I am using VS Code version 1.50.1 ..  My understanding is that I don't need any extension here, as your functionality is baked into VSCode itself, right?

I have also disabled all extensions and my VSCode is pretty much the default settings on the machine I am right now using.  And it is still showing the problem. 

I assume the extension of your file is .cpp?    Figured I'd ask, just in case..  If there are any other settings I should try tweaking, just let me know .. Thanks!
